content/docs/how-we-work/baseline/_index.md

```diff
@@ -191,6 +191,81 @@ Follow these steps only if the project doesn't already exist.
 {{< /tab >}}
 {{< /tabs >}}
 
+### Handling Alerts
+
+#### Monitoring Alerts in Sentry
+
+1. Access the Issues Dashboard
+
+    * In Sentry, go to the **Issues** tab to view all logged errors and alerts. The Issues dashboard provides a comprehensive overview of all logged events, grouped by the type and frequency of occurrence.
+
+2. Set Up Alert Rules:
+
+   * Configure alert rules to receive notifications for specific issues. Navigate to **Project Settings** > **Alerts** and define the conditions for alerts. You can set up alerts based on various criteria such as event frequency, error severity, and affected users.
+
+   * **Notification Settings**: Customize how and where you receive alerts. Options include email, SMS, Slack, or other third-party integrations. Make sure to configure these settings to ensure timely notifications.
+   * **Alert Thresholds**: Define thresholds for different alert types. For example, you might set a threshold to alert you only if an error occurs more than 10 times in an hour, helping to avoid alert fatigue.
```

Comment:
This is already covered in the documentation. Instead, we should explain what kind of alert rules they should set up. Please write instructions to send alerts when an issue is seen more than 30 times per hour. They can create more alerts as they want.